Mathlib/Data/Matrix/Defs.lean

```diff
@@ -497,4 +497,67 @@ abbrev subDownLeft {d u l r : Nat} (A : Matrix (Fin (u + d)) (Fin (l + r)) α) :
     Matrix (Fin d) (Fin l) α :=
   subDown (subLeft A)
 
+section RowCol
+
+/-- For an `m × n` `α`-matrix `A`, `A.row i` is the `i`th row of `A` as a vector in `n → α`.
+`A.row` is defeq to `A`, but explicitly refers to the 'row function` of `A`
+while avoiding defeq abuse and noisy eta-expansions,
+such as in expressions like `Set.Injective A.row` and `Set.range A.row`.
+(Note 04-07-2025 : the identifier `Matrix.row` used to refer to a matrix with all rows equal;
```

Comment:
Let's use a real date format like `2025-07-04` :)

Or perhaps just mention the major release where the old name was used (1.18.0)

Reply:
If we use the same date format as in deprecation notes, I think it should be `2025-04-07`.

Reply:
Yeah https://en.wikipedia.org/wiki/ISO_8601 is the only way.